packages/aws-rfdk/lib/deadline/lib/usage-based-licensing.ts

```diff
@@ -564,7 +564,7 @@ export class UsageBasedLicensing extends Construct implements IGrantable {
     this.service = new Ec2Service(this, 'Service', {
       cluster: this.cluster,
       taskDefinition,
-      desiredCount: props.desiredCount,
+      desiredCount: props.desiredCount ?? 1,
```

Comment:
Remind me... what was the CDK change that required this change?

Reply:
The link and short description is in the description of this PR. There were 2 options available:
1. In RFDK we just default to size 1 if `desiredCount` is not set explicitly. (what I did in this PR)
2. Do not set any default in RFDK for `Ec2Service`. In this case CDK will not add `DesiredCount` property to the CloudFormation template. But then it's not clear what to use for `minCapacity` and `maxCapacity` in ASG here https://github.com/aws/aws-rfdk/blob/mainline/packages/aws-rfdk/lib/deadline/lib/usage-based-licensing.ts#L508, that's why I decided to go with a simpler option 1.